### PR TITLE
SAK-28971; site copy does not map direct url

### PR DESCRIPTION
**Pull request by clhedrick**

### CODE REVIEW

File: shortenedurl/impl/src/java/org/sakaiproject/shortenedurl/impl/BitlyUrlService.java [hunk 1]
```diff
@@ -179,4 +179,12 @@ public void setApiKey(String apiKey) {
...
 	}
 
+	/*
+	 * this is a placeholder until someone who understands bitly decides what the
+	 * right approach is
+	 */
+	public boolean shouldCopy(String url) {
+		return false;
+	}
+
```
Comment (steveswinsburg) [inline shortenedurl/impl/src/java/org/sakaiproject/shortenedurl/impl/BitlyUrlService.java]: IMO its fine for URLs to no longer resolve in this particular case, implementors can handle it with an appropriate error page. Though a note should be added to the default.sakai.properties about this particular issue.

**Comment by steveswinsburg** [comment]
Hi Chuck, if you can update the default.sakai.properties with a note about the bitly urls not being copied/translated, then I will merge this. cheers.

**Comment by clhedrick** [comment]
Sure. Will do

**Comment by clhedrick** [comment]
I did the change to default.sakai.properties. Sorry for doing it the wrong place.

**Comment by buckett** [comment]
If you checkout the your local branch, make the change to default.sakai.properties and commit it, push it to your branch in github. You should then see that change appear in the files tab of this pull request. Then if you make a comment in the pull request Steve should be able to see it and merge the pull request.

Happy developing :cake:

**Comment by steveswinsburg** [comment]
Just checking if the default.sakai.properties is in already? (doesnt appear in this branch)

**Comment by clhedrick** [comment]
Yes. I added a comment

**Comment by steveswinsburg** [comment]
Yeah I meant the commit for it, not the comment :) All good.

**Pull request merged**